SingularityUI/app/actions/log.es6

```diff
@@ -183,21 +183,26 @@ export const taskGroupFetchNext = taskGroupId =>
     }
 
     dispatch({taskGroupId, type: 'LOG_REQUEST_START'});
-    let promises = tasks.map(function({taskId, exists, maxOffset, path, initialDataLoaded}) {
+    const promises = tasks.map(({taskId, exists, maxOffset, path, initialDataLoaded}) => {
       if (initialDataLoaded && exists !== false) {
-        let xhr = fetchData(taskId, path, maxOffset, logRequestLength);
-        return xhr.done(function({data, offset, nextOffset}) {
+        const xhr = fetchData(taskId, path, maxOffset, logRequestLength);
+        const promise = xhr.done(function({data, offset, nextOffset}) {
           if (data.length > 0) {
             nextOffset = offset + data.length;
             return dispatch(taskData(taskGroupId, taskId, data, offset, nextOffset, true, maxLines));
           }
         });
-      } else {
-        return Promise.resolve(); // reject("initialDataLoaded is false for task #{taskId}")
+        promise.taskId = taskId;
+        return promise;
       }
+      return Promise.resolve(); // reject("initialDataLoaded is false for task #{taskId}")
```

Comment:
The comment appears in a few other places in that action. I don't know the original purpose, and I'd rather not delete it in one place without deleting it in the other places (unless it makes sense to).
cc @tpetr